src/service/chain_rewardable_entities.proto

```diff
@@ -104,33 +108,38 @@ message re_reward_recipient_info {
 // all edge cases need to be covered
 // - reward recipients are updated
 // all edge cases need to be covered
-message mobile_hotspot_change_req_v1 {
+message mobile_atomic_hotspot_v1 {
   // Block height when this data was sourced
   uint64 block_height = 1;
 
   // Public key of the hotspot
-  pub_key pub_key = 2;
+  helium_pub_key pub_key = 2;
+
+  // Asset of the hotspot
+  solana_pub_key asset = 3;
 
   // Hotspot metadata
-  mobile_hotspot_metadata metadata = 3;
+  mobile_hotspot_metadata metadata = 4;
 
   // Controlling ownership of the NFT
-  re_controller_info controlling_ownership = 4;
+  re_controller_info owner = 5;
 
-  // Reward recipients for the NFT
-  repeated re_reward_recipient_info recipients = 5;
+  uint64 lifetime_claimed_rewards = 6;
```

Comment:
can drop this - feels more like "stat" information about the behavior of a hotspot.

Reply:
Erring on the side of more information -- you will want this in the future, and you're not going to have a great way to get it if we're not publishing it here.